source/robots.txt

```diff
@@ -0,0 +1,3 @@
+User-agent: *
+
+Sitemap: https://docs.advntr.dev/sitemap.xml
```

Comment:
hm, i wonder if we can template in the baseurl here

Reply:
didnt find a lead in the docs, so no clue.
also, fun fact: sphinx itself doesn't build the robots.txt using sphinx...